projects/nim-lang.org/package.yml

```diff
@@ -0,0 +1,46 @@
+distributable:
+  url: https://nim-lang.org/download/nim-{{ version }}.tar.xz
+  strip-components: 1
+
+# FIXME: test more versions
+versions:
+  - v1.6.14
+
+provides:
+  - bin/nim
+  - bin/nim_dbg
+  - bin/testament
+  - bin/nimsuggest
+  - bin/nimgrep
+  - bin/nim-gdb
+  - bin/atlas
+  - bin/nimpretty
+  - bin/nimble
+
+dependencies:
+  gnu.org/gcc: '*'
+  pcre.org: '*'
+  openssl.org: '*'
+
+build:
+  dependencies:
+    gnu.org/gcc: '*'
+    pcre.org: '*'
+    openssl.org: '*'
```

Comment:
if these are the same then you don't need to specify them again in the `build` node, however are these actually also runtime deps?